src/utils/UndoManager.js

```diff
@@ -316,7 +316,7 @@ export class UndoManager extends Observable {
    *
    */
   stopCapturing () {
-    this.lastChange = 0
+    this.lastChange = Number.NEGATIVE_INFINITY;
```

Comment:
There's a line above in this file that says `now - this.lastChange < captureTimeout`.

Setting `lastChange` to `0` will only work if the `captureTimeout` make `now - this.lastChange` equal to `now` which is the unix timestamp. I hope that's enough to show how #431 arises.

Reply:
god pls

Reply:
this pr now is not havn't process?